### ISSUE TITLE
Deployment request

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-event-landingpage#69]
I request a deployment of the code base of this repository to the actual production environment, after the following actions have been performed
* [x] PR #67 has been merged
* [x] Issue #68 has been fixed

**Comment by MikeMcC399** [comment]
It's difficult to see what has happened, however the website was definitely published sometime between Feb 8 and Jul 26, 2022.
 
- PR #40 was merged on Feb 8, 2022
- PR #51 was merged on Jul 26, 2022 by @thomasaugsten 

Most of the PRs after #51 are concerned with dependency updates or changes to the README file. https://github.com/corona-warn-app/cwa-event-landingpage/pull/67 will be the first new PR to change the appearance of the website again.

**Comment by Ein-Tim** [comment]
All tasks have been completed. The page can be deployed now.

**Comment by dsarkar** [comment]
@Ein-Tim Thanks, request to deploy has been forwarded.

**Comment by MikeMcC399** [comment]
So far, no change visible on https://s.coronawarn.app/en/. Is there a time-plan for deployment?

It looks as though the following PRs are related to resolving this issue:

- PR https://github.com/corona-warn-app/cwa-event-landingpage/pull/72
- PR https://github.com/corona-warn-app/cwa-event-landingpage/pull/74

**Comment by dsarkar** [comment]
Good morning @MikeMcC399, I will check!

**Comment by dsarkar** [comment]
@MikeMcC399 @Ein-Tim websites have been deployed.

**Comment by Ein-Tim** [comment]
Thanks for the information @dsarkar. Deployment confirmed. Closing this issue now.
